docs/v0.3/partials/_install-sdk-go.md

```diff
@@ -0,0 +1,11 @@
+:::note
+The Dagger SDK for Go requires [Go 1.15 or later](https://go.dev/doc/install).
+:::
+
+Install the Dagger SDK for Go using the commands below:
+
+```shell
+go get go.dagger.io/dagger@v0.3.0-alpha.1
```

Comment:
This is implemented as a partial so it can be reused. Maybe we can add it to our release process so this always reflects the latest tag.

Reply:
I think once we get to GA (or whenever the Go SDK gets a stable release), we can remove the version specifier since the latest tag will be automatically pulled
https://go.dev/doc/modules/release-workflow#pre-release